tests/samples/recursion.py

```diff
@@ -18,45 +18,49 @@ def main():
 expected_output = '''
 Source path:... Whatever
 Starting var:.. x = 4
-20:28:17.875295 call         5 def factorial(x):
-20:28:17.875509 line         6     if x <= 1:
-20:28:17.875550 line         8     return mul(x, factorial(x - 1))
+09:31:32.691599 call         5 def factorial(x):
```

Comment:
This is not ideal, it ruins the git history.

Reply:
OK. I'll modify and rebase it.